cli/crud/import/containers.go

```diff
@@ -0,0 +1,163 @@
+// This file contains types for defining containers (structs) that store
+// contexts during crud import work.
+
+package crudimport
```

Comment:
Why do we need two packages (`crudimport` and `crudexport`)? I propose to just have a package `crud`.